frontend/src/app/middleware/middleware.service.ts

```diff
@@ -80,20 +80,29 @@ export class MiddlewareService {
       .post(`${API_BASE}/claims`, claim, {
         headers: this.headers,
       })
+      .subscribe( 
+        data => {},
+        error => {},
+        () => this.router.navigate([`/claims/${claim.vehicle.vin}`])
+      );
+  };
+
+  editClaim(claim: DeepPartial<Protos.Claim>) {
+    console.log(claim);
+    return this.http
+      .post(`${API_BASE}/claims/${claim.vehicle.vin}`, claim, {
+        headers: this.headers
+      })
       .subscribe(
-        success => {
-          // Log when successful / errored navigation
-          this.router.navigate([`/claims/${claim.vehicle.vin}`]).then(
-            nav => {
-              console.log(nav);
-            },
-            error => {
-              console.log(error)
-            }
-          );
+        data => {},
+        error => {alert("Edit failed, please check console log")},
```

Comment:
use snackbar thing with a helpful message rather than window.alert.

Reply:
Snackbar will take some finagling to get functional. This is fine for now.

Reply:
avoiding anything snackbar for now, that is a @sloan-seamus task to implement.